### ISSUE TITLE
some questions about MSCOCO Multi-label Classification

### ISSUE THREAD

**Issue by mymuli** [allenai/elastic#7]
Hi， Thank you for sharing the good idea and code.
But I have some questions about MSCOCO Multi-label Classification
1. What do these two files **coco_resnext50.pth.tar** and **coco_resnext50_elastic.pth.ta**r mean respectively?

2. Are these two files the result of **ImageNet** pre-training or running on **MSCOO**?

3. If I want to do the **ResNeXt50 + Elastic** experiment from scratch, which file should I use?

**Comment by csrhddlam** [comment]
Hi,

Thanks for your interest in our work.

coco_*.pth.tar are models that have already been trained on COCO (pre-trained on ImageNet before being trained on COCO).

If you want to reproduce the COCO experiment only, from ImageNet pre-trained models, you should download ImageNet models described in ImageNet Classification, i.e. http://ai2-vision.s3.amazonaws.com/elastic/imagenet_models/resnext50_elastic.pth.tar in the case of ResNeXt50 + Elastic.

If you want to do ResNeXt50 + Elastic experiment literally from scratch, you should train first on ImageNet from random initialization, save your own checkpoint, and then finetune on COCO.

Best,
Huiyu

**Comment by mymuli** [comment]
@csrhddlam  
 Thank you very much for your reply, which solved my problem！
